tls/resource_private_key.go

```diff
@@ -10,6 +11,7 @@ import (
 	"fmt"
 
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/mikesmitty/edkey"
```

Comment:
Is this dependency necessary?  It's quite small, hasn't been updated since 2017, and could perhaps be inlined in some form into https://github.com/terraform-providers/terraform-provider-tls/blob/master/tls/util.go.  Given the sensitive nature of this provider, it's nice that the [dependency list](https://github.com/terraform-providers/terraform-provider-tls/blob/master/go.mod) is so small.

Reply:
Right. Since https://github.com/mikesmitty/edkey is MIT licensed, I'll just add it into the utils :)

Reply:
Done. FYI: @mikesmitty :)

Reply:
Thanks for the tag, I'm glad to see it be useful